api_client/python/grr_api_client/flow.py

```diff
@@ -105,6 +106,14 @@ def GetCollectedTimeline(
         client_id=self.client_id, flow_id=self.flow_id, format=fmt)
     return self._context.SendStreamingRequest("GetCollectedTimeline", args)
 
+  def GetOsqueryResults(
+      self,
+      fmt: osquery_pb2.ApiGetOsqueryResultsArgs.Format
```

Comment:
Missing trailing comma. Without the internal auto-formatter can fold it in awkward ways.